apis/v1alpha1/httproute_types.go

```diff
@@ -280,8 +280,9 @@ type HTTPPathMatch struct {
 
 	// Value of the HTTP path to match against.
 	//
-	// +kubebuilder:validation:MinLength=1
-	Value string `json:"value"`
+	// +optional
+	// +kubebuilder:default="/"
+	Value string `json:"value,omitempty"`
```

Comment:
Since this is now optional I think it also needs to be a pointer. (cc @danehans to be sure)

Reply:
+1

Reply:
Updated. Thank you!